### ISSUE TITLE
Can the project do back propagation, i.e., loss.backward()?

### ISSUE THREAD

**Issue by Yusifu** [Tiiiger/bert_score#12]
I have a domain corpus, which have the binary golden annotation: sentence pair A-B is similar, sentence pair C-D is not similar.

Can I use this corpus to fine-tune bert and get more accurate sentence similarity score based on the corpus?
How to define the loss function and do loss.backward?
And how is the performance on with that loss function?
Have you ever had a try?

**Comment by felixgwu** [comment]
Hi @Yusifu,

We do not fine-tune BERT since it may not generalize to new corpora.
If you would like to fine-tune BERT, we suggest you follow this example in [huggingface/pytorch-pretrained-BERT](https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/examples/run_classifier.py).
I believe that your task is quite similar to the MRPC dataset in GLUE benchmark.

Best,
Felix

**Comment by Tiiiger** [comment]
I am closing this. feel free to reopen if you have any other questions.
